README.md

```diff
@@ -24,7 +24,6 @@ truffle build
 
 To test the project:
 ```
-npm install chai ganache-cli
```

Comment:
Don't need to launch ganache-cli separately since we used `truffle run coverage`

Reply:
no what if you want to debug some specific transaction?

Reply:
got it